scripts/create-environment.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+if [ -d "$HOME/.osmosisd" ]; then
+  echo "OSMOSISD_ENVIRONMENT=mainnet" > $HOME/.osmosisd/.env
+fi
```

Comment:
1. What if .osmosisd exists but .osmosisd/.env does not?
2. If `"$HOME/.osmosisd"` does not exist, can we create one?

Reply:
This step runs after installing binary so `$HOME/.osmosisd` is required. If not, it can be considered a problem with the installation step.
Also, line 4 will create a `.env` file and fill it.

Reply:
If you have any suggestions pls lmk!

Reply:
I see. Can we echo an error message if `"$HOME/.osmosisd"` does not exist? So that if this installation problem does happen, a user can know where to start debugging.

Also, would `echo "OSMOSISD_ENVIRONMENT=mainnet" > $HOME/.osmosisd/.env` append `"OSMOSISD_ENVIRONMENT=mainnet"` to `.env` every time we run `make build`? We should make sure that we don't append to the file redundantly on every build

Reply:
Hrmm, I'd rather we do this in the go code, similar to where we currently init the configs

I don't feel great about adding a separate process for this file being initialized

Reply:
I think so?

I always have to re-investigate every time where they get initialized, since actually multiple commands initialize the client.toml